### PR TITLE
update tutorial with epidemics 0.2.0

### LINKED ISSUE #3: remove `introduction.Rmd` episode
let's keep this in the [tutorials repo](https://epiverse-trace.github.io/tutorials/) as part of the Outbreak analytics overview

### LINKED ISSUE #10: update epidemics to release 0.2.0

### PR DESCRIPTION
**Pull request by avallecam**
Fix #10 
Fix #3 

This PR updates the code API after the latest epidemics release. 

- This edits the code and text mentioning functions and arguments. 
- This also updates the `renv. lock` file
- We also removed the intro episode. We expect that the automatic update next month fix the dependencies of this episode.
- We kept one vaccination hint hidden to later evaluate whether to #12 

**how to review?**

- To quickly diagnose the content, you can read online the knitted content of this PR in branch [`md-output-PR-11`](https://github.com/epiverse-trace/tutorials-late/tree/md-outputs-PR-11)
  - [simulating-transmission](https://github.com/epiverse-trace/tutorials-late/blob/md-outputs-PR-11/simulating-transmission.md)
  - [model-choices](https://github.com/epiverse-trace/tutorials-late/blob/md-outputs-PR-11/model-choices.md)
  - [modeling-interventions](https://github.com/epiverse-trace/tutorials-late/blob/md-outputs-PR-11/modelling-interventions.md)
  - [compare-interventions](https://github.com/epiverse-trace/tutorials-late/blob/md-outputs-PR-11/compare-interventions.md)
- To render this locally, follow the steps in the [reviewer role section of the contributing.md file](https://github.com/epiverse-trace/tutorials-late/blob/main/CONTRIBUTING.md#reviewer). If local rendering doesn't work, please, report them in Issues.

### CODE REVIEW

**Comment by avallecam** [comment]
all edits are now finished - just fixed an overlooked error in https://github.com/epiverse-trace/tutorials-late/pull/11/commits/a005aa69119d621c73d040beb6aff37dc198a38a

**Review by pratikunterwegs — APPROVED** [review]
Thanks @avallecam - only a few comments, but looks good otherwise.

File: episodes/compare-interventions.Rmd [hunk 1]
```diff
@@ -135,21 +135,23 @@ DiagrammeR::grViz("digraph{
...
 2. Using the output, plot the number of deaths through time
 
-
+<!--
 ::::::::::::::::: hint
 
 ### Vaccination code 
 
-To run the model with no vaccination in place we can *either* create two vaccination objects (one for each dose) using `vaccination()` with the time start, time end and vaccination rate all set to 0, or we can use the `no_vaccination()` function to create a vaccination object for two doses with all values set to 0.
+To run the model with no vaccination in place we can *either* create two vaccination objects (one for each dose) using `vaccination()` with the time start, time end and vaccination rate all set to 0.
```
[the remark this answers is not in the document]
Comment (avallecam) [reply episodes/compare-interventions.Rmd]: yes, indeed. for that reason this is between `<!-- ... -->` from 138 to 154, but this is not too visible. I keep it to evaluate if we can make the challenge to include vaccination as part of the task in #12. Added the blocks in each line for visibility https://github.com/epiverse-trace/tutorials-late/pull/11/commits/f9f96c1403f48e6bac42a2bd6d7f6e25b38c5df9

File: episodes/introduction.Rmd
[the remark this answers is not in the document]
Comment (avallecam) [reply episodes/introduction.Rmd:1]: first, sorry for mixing issues in this PR. This fixes #3. this lives in the `tutorials` repo and we aim to keep it there https://github.com/epiverse-trace/tutorials/pull/137. we have a pending issue to add a specific intro for this set of tutorials in #5

File: episodes/model-choices.Rmd [hunk 2]
```diff
@@ -104,7 +104,7 @@ Finally, interventions such as vaccination may be of interest. A model may or ma
 ## Available  models in `epidemics`
  
 The R package `epidemics` contains functions to run existing models.
-For details on the models that are available, see the package [vignettes](https://epiverse-trace.github.io/epidemics/articles). To learn how to run the models in R, read the documentation using `?epidemics::model_ebola_r`. 
+For details on the models that are available, see the package [vignettes](https://epiverse-trace.github.io/epidemics/articles). To learn how to run the models in R, read the documentation using `?epidemics::model_ebola`.
```
[the remark this answers is not in the document]
Comment (avallecam) [reply episodes/model-choices.Rmd]: Good point. I redirected the links to more specific sites. 
- To find what models are available, go to the reference guide where `model_*()` functions are on top. 
- To learn how to run, go to vignettes in the section "guide to lib models"

Added in https://github.com/epiverse-trace/tutorials-late/pull/11/commits/b7cf6714af9211777646cc731e457ea0a7596162

File: episodes/modelling-interventions.Rmd [hunk 3]
```diff
@@ -140,9 +140,9 @@ The contacts within group 1 are reduced by 50% twice to accommodate for a 50% re
 Using transmissibility $= 2.7/5.5$ (remember that [transmissibility = $R_0$/ infectious period](../episodes/simulating-transmission.md#the-basic-reproduction-number-r_0)), infectiousness rate $1/= 4$ and the recovery rate $= 1/5.5$ we run the model with` intervention = list(contacts = close_schools)` as follows :
```
[the remark this answers is not in the document]
Comment (avallecam) [reply episodes/modelling-interventions.Rmd]: thanks, covered in https://github.com/epiverse-trace/tutorials-late/pull/11/commits/a005aa69119d621c73d040beb6aff37dc198a38a

File: episodes/modelling-interventions.Rmd [hunk 4]
```diff
@@ -232,9 +232,9 @@ mask_mandate <- intervention(
 To implement this intervention on the parameter $\beta$, we specify `intervention = list(beta = mask_mandate)`.
```
[the remark this answers is not in the document]
Comment (avallecam) [reply episodes/modelling-interventions.Rmd]: Changed in https://github.com/epiverse-trace/tutorials-late/pull/11/commits/f4ac528177c6dddf1804be05b68c1301b1275325

File: episodes/modelling-interventions.Rmd [hunk 5]
```diff
@@ -303,7 +303,7 @@ To implement both contact and rate interventions in the same simulation they mus
 
 Pharmaceutical interventions (PIs) are measures such as vaccination and mass treatment programs. In the previous section, we assumed that interventions reduced the value of parameter values while the intervention was in place. In the case of vaccination, we assume that after the intervention individuals are no longer susceptible and should be classified into a different disease state. Therefore, we specify the rate at which individuals are vaccinated and track the number of vaccinated individuals through time. 
 
-The diagram below shows the SEIRV model implemented using `model_default_cpp()` where susceptible individuals are vaccinated and then move to the $V$ class.
+The diagram below shows the SEIRV model implemented using `model_default()` where susceptible individuals are vaccinated and then move to the $V$ class.
```
[the remark this answers is not in the document]
Comment (avallecam) [reply episodes/modelling-interventions.Rmd:306]: Fixed in https://github.com/epiverse-trace/tutorials-late/pull/11/commits/61ac141715c9c6f7648e8bc419d2002d115a6719

File: episodes/model-choices.Rmd [hunk 6]
```diff
@@ -186,7 +186,7 @@ DiagrammeR::grViz("digraph {
 The model is capable of predicting an Ebola type outbreak, but as the model is deterministic, we are not able to explore stochastic variation in the early stages of the outbreak.
```
[the remark this answers is not in the document]
Comment (avallecam) [reply episodes/model-choices.Rmd]: good point. Changed this to "simulate" or "trajectories" instead in https://github.com/epiverse-trace/tutorials-late/pull/11/commits/0261a31089188a87ebf6cae0ba677e5ebb93e83f

File: episodes/modelling-interventions.Rmd (hunk 5 above)
[the remark this answers is not in the document]
Comment (avallecam) [reply episodes/modelling-interventions.Rmd:306]: I recovered them at https://github.com/epiverse-trace/tutorials-late/pull/11/commits/f5e2d2eed425b7bee3894ba598d90681ff85e806. 
- When we keep the `vaccination = NULL` then we would be using `model_default()` for a SEIR?
- In the second one, the writing is referring to the SEIR components of the SEIRV model, which is accurate.

**Comment by avallecam** [comment]
Thanks for the review and comments @pratikunterwegs Appropriate edits suggestions, commits were added to each. Replies to your questions:

> The whole file has been deleted - where does the introduction live now?

I mixed issues loosing the specific scope of this! This is also fixing #3 more in https://github.com/epiverse-trace/tutorials-late/pull/11#discussion_r1535770100

> A few lines above this is called an SEIR model, would be good to change that.

This is used in the context of non-pharma interventions or to refer to the SEIR definition in first episode. Full reply in https://github.com/epiverse-trace/tutorials-late/pull/11#discussion_r1535900891

I'll merge this after passing checks.

**Pull request merged**